mantidproject/mantid
# Release Notes 5.0 - Index and Core

**Description of work.**
Tidied up release notes, the main page and core (plot and WB) pages.
Deleted the UI page as it was rather bare (moved that info elsewhere)

**To test:**

Build docs and check formatting of bullet points and images and that links work.

Fixes #28235 
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
